index.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>LinuxChix India</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

Comment:
It is not my website :) I'm just maintaining this project.

I'm not sure why not allow users to zoom in / out, so I asked if you had something specific in mind for this. 

May be @shivan1b can have a say here.